src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/Marshalling/NullMarshallingGeneratorFactory.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Interop
+{
+    public sealed class NullMarshallingGeneratorFactory : IMarshallingGeneratorFactory
```

Comment:
I don't understand the null name here. Seems like it should be called `UnsupportedMarshallingGeneratorFactory`.

Reply:
I was following the "Null Object Pattern" as I didn't have a better name in mind when I wrote this. I'll change the name (yours is better)